app/src/main/java/com/medmapper/v33001/dao/MedicineDAO.kt

```diff
@@ -7,7 +7,7 @@ import kotlinx.coroutines.flow.Flow
 @Dao
 interface MedicineDAO {
     @Query("SELECT * FROM Medicine ORDER BY name ASC")
-    fun getAlphabetizedMedicine(): Flow<List<Medicine>>
+    fun getAllAlphabetizedMedicine(): Flow<List<Medicine>>
```

Comment:
Just added All to the name of the function, as the query is fetching all rows from the Medicine DB